packages/fxa-settings/src/pages/Signin/index.tsx

```diff
@@ -0,0 +1,244 @@
+    entrypoint_variation: 'react',
+  });
+
+  const isPocketClient = serviceName === 'Pocket';
+  const [error, setError] = useState('');
+  const [password, setPassword] = useState('');
+  const ftlMsgResolver = useFtlMsgResolver();
+  const localizedPasswordFormLabel = ftlMsgResolver.getMsg(
+    'password',
+    'Password'
+  );
+  const localizedPocketAriaLabel = ftlMsgResolver.getMsg(
+    'pocket-logo',
+    'Pocket logo'
+  );
+  const localizedLogoAriaLabel = ftlMsgResolver.getMsg(
+    'service-logo',
+    `${serviceName} logo`,
+    { serviceName }
+  );
+
+  const signInUsingLoggedInAccount = useCallback(() => {
+    // TODO: add in functionality to sign in using the logged in account
+    // return an error to be displayed if anythign goes wrong.
+  }, []);
+
+  const signInWithPassword = useCallback((email: string, password: string) => {
+    // TODO: add in the functionality to actually sign a user in using their password
+    // return an error to be displayed if anything goes wrong.
+  }, []);
+
+  // TODO: This page is also supposed to render the user card, complete with avatar.
+  // The avatar mixin is complex enough that I think it deserves its own ticket.
+
+  // TODO: The user-card mixin does some fancy footwork listening to see if you access token changes
+  // because it's possible for the access token to be invalidated when we display the avatar image.
+  // if we get the notification of a change, we re-render the card.
+
+  const onSubmit = useCallback(async () => {
+    try {
+      isPasswordNeeded
+        ? signInWithPassword(email, password)
+        : signInUsingLoggedInAccount();
+      // TODO: add message in Alert Bar re: success, then navigate to where appropriate
+    } catch (e) {
+      // TODO: metrics event for error
+      // TODO: Add in localized Alert Bar message for error
+      setError(e);
+    }
+  }, [
+    email,
+    password,
+    signInUsingLoggedInAccount,
+    signInWithPassword,
+    isPasswordNeeded,
+    setError,
+  ]);
+
+  // TODO:
+  // Add in the Alert Bar in place of the original `success` and `error` display divs
+
+  return (
+    <>
+      <div className="mb-4">
+        {isPasswordNeeded ? (
+          <FtlMsg id="signin-password-needed-header">
+            <h1 className="card-header">
+              Enter your password{' '}
+              <span className="card-subheader">for your Firefox account</span>
+            </h1>
+          </FtlMsg>
+        ) : (
+          <>
+            <FtlMsg id="signin-header">
+              <h1 className="card-header">Sign in</h1>
+            </FtlMsg>
+            <>
+              {isPocketClient && (
+                <FtlMsg id="signin-subheader-with-logo">
+                  <span className="card-subheader">
+                    Continue to{' '}
+                    <span>
+                      <PocketLogo
+                        className="inline"
+                        aria-label={localizedPocketAriaLabel}
```

Comment:
This is a weird one where I kind of think we should explicitly call this "Pocket" (which wouldn't require l10n) rather than "Pocket logo" because we're using it in a sentence like, "Continue to Pocket" instead of "Continue to Pocket logo". Alternatively, we could use an `aria-hidden="true"` on this and have a `sr-only` class that reads "Pocket". I don't feel super strongly though.

Also, does `PocketLogo` need to be in a `span`?